tests/tasks/teardown/awscli-eks.yaml

```diff
@@ -38,6 +38,7 @@ spec:
           aws eks wait nodegroup-deleted --nodegroup-name $i --cluster-name $(params.cluster-name) $ENDPOINT_FLAG --region $(params.region);
       done;
       aws eks delete-cluster --name $(params.cluster-name) --region $(params.region) $ENDPOINT_FLAG
+      sleep 900
```

Comment:
This is a lot of time like 15mins ? Do we really need this ? can we see if we have an option of `wait until` etc through `aws eks cli` to ensure it's deleted instead of waiting arbitrarily ?